src/App.css

```diff
@@ -73,43 +85,49 @@ span.empty {
     background: #fff; }
 
 .group-0 {
-  box-shadow: inset 0px -1px 9px 1px #ccc; }
+  box-shadow: inset 0px 0px 9px 1px #ccc; }
```

Comment:
This one is different from the other groups, is that intentional?

Reply:
Yes, it's the sort of misc category, so the color is somewhat more muted